code/game/camera_manager/camera_manager.dm

```diff
@@ -191,7 +200,7 @@
 	// Cameras that get here are moving, and are likely attached to some moving atom such as cyborgs.
 	last_camera_turf = get_turf(cam_location)
 
-	var/list/visible_things = current.isXRay() ? range(current.view_range, cam_location) : view(current.view_range, cam_location)
+	var/list/visible_things = current.isXRay() ? range("[target_width]x[target_height]", cam_location) : view("[target_width]x[target_height]", cam_location)
```

Comment:
This change seems to just produce incorrect results in every view:
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/414b2e1c-1d95-463e-a09e-584dfaf70264)
Despite it having a 7 and 7 for width and height, it made a view of 5x7.

If its not merely because you don't have the fix setting target_width and target_height in `/datum/component/camera_manager/proc/set_camera_rect(source, x, y, z, w, h)` like in #5298 then I'm not sure.

Reply:
I also recommend using 515.1609+ so you don't have the viewport sizing bug.

Reply:
Reverting only that line the views seem to be properly displayed: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/d1993da2-e7b3-4283-9abe-f946c5cd77d5)